### ISSUE TITLE
[Bug][SF][1.29.0][List][A11y][Expected Fixed Time: ASAP] The status violates the contrast demand when hovering on the list in high contrast mode

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#9869]
### Describe the bug

The status violates the contrast demand when hovering on the list in HCB mode.

code example

```
import React, { useCallback, useMemo } from 'react';

import { List, StandardListItem } from '@ui5/webcomponents-react';
function App() {
  const data = [
    {
      id: 1,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 2,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 3,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 4,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 5,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 6,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 7,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 8,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 9,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 10,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 11,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 1,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 2,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 3,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 4,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 5,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 6,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 7,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 8,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 9,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 10,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 11,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 1,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 2,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
    {
      id: 3,
      link: 'www.google.com',
      name: 'Notebook Basic',
      price: '956EUR',
      dimension: '30 x 18 x 3cm',
    },
  ];
  return (
    <>
      <List growing="None" headerText="List with ListItemStandard">
        {data.map((x) => {
          return (
            <StandardListItem
              additionalText="www"
              additionalTextState="Warning"
            ></StandardListItem>
          );
        })}
      </List>
    </>
  );
}

export default App;
```

### Isolated Example

https://stackblitz.com/edit/github-xbtmwe-zgqwey?file=src%2FApp.tsx,src%2Fmain.tsx,node_modules%2F%40ui5%2Fwebcomponents-react%2Fdist%2Fcomponents%2FThemeProvider%2Findex.d.ts

### Reproduction steps

1. use HCB mode
2. hover on the list item
3. use CCA to test.

### Expected Behaviour

No violation

### Screenshots or Videos

<img width="521" alt="image" src="https://github.com/user-attachments/assets/c734ddb1-02dc-40fc-a4c9-7c15357f36d8">

### UI5 Web Components for React Version

1.29.0

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Organization

SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @ui5-webcomponents-topic-p I am forwarding this issue to you, because the contrast ration between the color of the additionalText in warning state and the color of the background of the list item when hovering in HCB mode is violated. Maybe this should also be discussed with design team.

Regards,
Iliana

**Comment by kgogov** [comment]
Hi @wilson-xu-sap,

This issue has been resolved with #10272.

Thank you for bringing this to our attention! 
If you encounter any further issues or have additional feedback, feel free to reach out.

Best regards,
Konstantin

**Comment by wilson-xu-sap** [comment]
> Hi @wilson-xu-sap,
> 
> This issue has been resolved with #10272.
> 
> Thank you for bringing this to our attention! If you encounter any further issues or have additional feedback, feel free to reach out.
> 
> Best regards, Konstantin

Glad to hear that, thanks!
